kie-server-parent/kie-server-services/kie-server-services-jbpm-queries/pom.xml

```diff
@@ -0,0 +1,176 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+	<parent>
+		<groupId>org.kie.server</groupId>
+		<artifactId>kie-server-services</artifactId>
+		<version>7.1.0-SNAPSHOT</version>
+	</parent>
+
+	<artifactId>kie-server-services-jbpm-queries</artifactId>
+
+	<name>KIE :: Execution Server :: Services :: jBPM Queries Extension</name>
+	<description>KIE jBPM Queries Execution Server Extension</description>
+
+	<dependencyManagement>
+		<dependencies>
+  		<!-- TODO: This should be moved to a parent BOM -->
+			<dependency>
+				<groupId>org.kie.server</groupId>
+				<artifactId>kie-server-jbpm-queries-api</artifactId>
+				<version>7.1.0-SNAPSHOT</version>
+			</dependency>
+		</dependencies>
+	</dependencyManagement>
+
+	<dependencies>
+		<dependency>
+			<groupId>org.kie</groupId>
+			<artifactId>kie-api</artifactId>
+		</dependency>
+		<dependency>
+			<groupId>org.kie</groupId>
+			<artifactId>kie-internal</artifactId>
+		</dependency>
+		<dependency>
+			<groupId>org.kie</groupId>
+			<artifactId>kie-ci</artifactId>
+		</dependency>
+
+		<dependency>
+			<groupId>org.kie.server</groupId>
+			<artifactId>kie-server-api</artifactId>
+		</dependency>
+		<dependency>
+			<groupId>org.kie.server</groupId>
+			<artifactId>kie-server-jbpm-queries-api</artifactId>
+		</dependency>
+		<dependency>
+			<groupId>org.kie.server</groupId>
+			<artifactId>kie-server-services-common</artifactId>
+		</dependency>
+
+		<dependency>
+			<groupId>org.kie.server</groupId>
+			<artifactId>kie-server-common</artifactId>
+		</dependency>
+    
+    <!-- TODO: This dependency is needed because we need access to ConvertUtils. Maybe it's an idea to put those utils in a common lib. -->
```

Comment:
fine with me to do that after this PR is merged, though remove the todo from code and maybe file jira instead